source/Octostache/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "0.0.0-*",
+  "dependencies": {
+    "Markdown": "2.0.0",
+    "NETStandard.Library": "1.6.0",
+    "Newtonsoft.Json": "9.0.1",
+    "Sprache": "2.0.1"
+  },
+  "frameworks": {
+    "net451": {
+      "frameworkAssemblies": {
+        "System.Core": "*",
+        "System.Runtime.Caching": "*",
+        "Microsoft.CSharp": "*"
+      },
+      "buildOptions": {
+        "define": [ "NET46" ]
```

Comment:
Should we standardise on NET40 (defining the runtime, not the framework)? It also doesn't match the actual framework in this case.